packages/a11y-testing/src/definitions/Pill/pillOptionBehaviorDefinition.ts

```diff
@@ -0,0 +1,8 @@
+import { Rule } from '../../types';
+import { BehaviorRule } from '../../rules/rules';
+
+export const pillBehaviorDefinition: Rule[] = [
```

Comment:
just suggestion if you would like to extent these test with testing Delete and Backspace?

```
  performDismiss: {
        keyCombinations: [{ keyCode: keyboardKey.Delete }, { keyCode: keyboardKey.Backspace }],
      },
```

Reply:
It's more complicated than it looks, we don't have `onClick` we have `onDismiss` so we would need to do some bigger change in the a11y-testing  I guess, let's skip this here for now